addons/l10n_it_edi_withholding/models/account_edi_format.py

```diff
@@ -11,6 +11,17 @@
 class AccountEdiFormat(models.Model):
     _inherit = 'account.edi.format'
 
+    def _l10n_it_edi_search_tax_for_import(self, company, percentage, extra_domain=None):
+        """ In case no withholding_type or pension_fund is specified, exclude taxes that have it.
+            It means that we're searching for VAT taxes, especially in the base l10n_it_edi module
+        """
+        domain_fields = [condition[0] for condition in extra_domain]
```

Comment:
Need to be careful because e.g. condition could be just '|'.  Maybe we can just add an optional param only_vat=True